.github/workflows/all_solutions.yml

```diff
@@ -215,16 +215,26 @@ jobs:
           if-no-files-found: error
 
       - name: Get Code Signing Certificate
+        if: ${{ github.event.release }}
         id: write_cert
         uses: timheuer/base64-to-file@v1
         with:
           fileName: 'newrelic_code_sign_cert.pfx'
           encodedString: ${{ secrets.SIGNING_CERT }}
           
       - name: Install Code Signing Certificate
+        if: ${{ github.event.release }}
         run: |
+          Write-Host "certutil.exe -f -user -p <passphrase> -importPFX ${{ steps.write_cert.outputs.filePath }} NoRoot"
           certutil.exe -f -user -p ${{ secrets.CERT_PASSPHRASE }} -importPFX ${{ steps.write_cert.outputs.filePath }} NoRoot
         shell: powershell
+      
+      - name: Create Self-signed code signing cert
+        if: ${{ github.event.pull_request }}
```

Comment:
This will cover pull requests, and the step that is gated by `github.event.release` takes care of release builds.  What happens for manual runs or merges to `main`?

Reply:
- Your right, I missed manual runs.  I'll add it.
- Merges to main only happen via the PR process.  We don't run any tests on merge anymore since we run all tests during the PR review phase.

Reply:
Got it, thanks